plugins/outputs/redistimeseries/redistimeseries_test.go

```diff
@@ -0,0 +1,26 @@
+package redistimeseries
+
+import (
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestConnectAndWrite(t *testing.T) {
```

Comment:
Do you think it makes sense to use the [redis mock package](https://github.com/go-redis/redismock) for testing without requiring a server?

Reply:
Can you please comment!?

Reply:
Any comment regarding my question

>  Do you think it makes sense to use the redis mock package for testing without requiring a server?